judge_target_object/BoundingBox2D bounding_box
int32[] x_masks
int32[] y_masks
